src/CMakeLists.txt

```diff
@@ -321,6 +331,28 @@ foreach(FN ${SHADERS})
     list(APPEND COPIED_FILES "${OUTFN}")
 endforeach()
 
+if(${INSTALL_SAMPLE_MESHES})
+    #Copy sample meshes files to distrib directory
+    if(NOT ${CMAKE_CURRENT_SOURCE_DIR}/distrib EQUAL ${MESHLAB_BUILD_DISTRIB_DIR} )
+        file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/distrib/sample 
+             DESTINATION ${MESHLAB_BUILD_DISTRIB_DIR} 
+             PATTERN "*/rangemaps" EXCLUDE )
+        if(${INSTALL_SAMPLE_RANGEMAPS} )
+            file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/distrib/sample/rangemaps DESTINATION ${MESHLAB_SAMPLE_OUTPUT_DIR} )
+        endif()    
+    endif()
+        
+    #Rules to install sample files from distrib/sample directory
+    install(
+        DIRECTORY ${MESHLAB_SAMPLE_OUTPUT_DIR}
+        DESTINATION ${MESHLAB_SAMPLE_INSTALL_DIR}
+        COMPONENT Samples)
+    file(GLOB_RECURSE SAMPLE 
+         LIST_DIRECTORIES false
+         ${MESHLAB_SAMPLE_OUTPUT_DIR}/* )
+    list(APPEND COPIED_FILES ${SAMPLE} )
```

Comment:
this isn't what the copied_files thing is for - it's for files that have received an `add_custom_command` to copy. and thus need a target to mention them to perform the copy.